package.json

```diff
@@ -83,6 +83,6 @@
     "preversion": "npm run build && (git diff --quiet || git commit -am build)"
   },
   "engines": {
-    "node": ">= 8.16.2"
+    "node": ">= 12"
```

Comment:
It's one thing to ensure support in CI but it's another thing to change support in a minor version. I understand that only LTS/latest node versions are supported but real functional changes should only happen during a major version, otherwise it's going to break a lot of people.

Reply:
Sorry, left this a while back but forgot to submit...

Reply:
If jsdoc wants to support node v8 they could import the es5 version of marked

```js
const marked = require('marked/lib/marked.js');
```

Reply:
For the record according to jsdoc [engines](https://github.com/jsdoc/jsdoc/blob/ac60926981d547688903eba505abd95422260602/package.json#L19-L21) field it only supports `>=v14.17.1`

Reply:
This change is breaking our CI build now. We use this through [Typedoc](https://github.com/TypeStrong/typedoc/blob/master/package.json), with that and our lib using Node `>= 10.18.0`. It would be great if you could revert this change. This is a breaking change as per semantic versioning. I'd recommend you do this in a v3.x release and have a deprecation period to allow dependents to update their packages or find some other mitigation.

```
error marked@2.1.1: The engine "node" is incompatible with this module. Expected version ">= 12". Got "10.18.0"
error Found incompatible module.
```

Reply:
@Fleker Node.js 10 reached end of life in April and is no longer supported https://nodejs.org/en/about/releases/

You can probably disable that error if you really need to with `yarn install --ignore-engines`